src/ssl.c

```diff
@@ -8207,7 +8211,14 @@ int wolfSSL_CertManagerEnableCRL(WOLFSSL_CERT_MANAGER* cm, int options)
     WOLFSSL_ENTER("wolfSSL_CertManagerEnableCRL");
     if (cm == NULL)
         return BAD_FUNC_ARG;
-
+#if defined(OPENSSL_COMPATIBLE_DEFAULTS)
+    if (options == 0) {
+        /* Turn off doing Leaf CRL check */
+        cm->crlEnabled = 0;
+        /* Turn off all checks */
+        cm->crlCheckAll = 0;
+    }
+#endif
     #ifdef HAVE_CRL
         if (cm->crl == NULL) {
             cm->crl = (WOLFSSL_CRL*)XMALLOC(sizeof(WOLFSSL_CRL), cm->heap,
```

Comment:
If OPENSSL_COMPATIBLE_DEFAULTS is defined, and we don't end up enabling CRL, is there any point in allocating and initializing the cert manager's CRL?  Or should the entire function be skipped in this case?  I'm not seeing anywhere else in the code where we set crlEnabled.

Reply:
@kareem-wolfssl. Excellent observation! Your feedback has been implemented in the update.